tests/cluster/check_remoteaccount.py

```diff
@@ -86,6 +93,18 @@ def check_wait_for_http_timeout(self):
             actual_timeout = time.time() - start
             assert abs(actual_timeout - timeout) / timeout < 1
 
+    def check_ssh_checker(self):
```

Comment:
I'm presuming other existing tests provide coverage for when there are no exception checks? Please make sure that we have covered the cases when there's more than one exception check, no exception checks or invalid exception check(s).

Reply:
No exception checks are the default case, so they are covered, will add the multiple case specificly